### PR TITLE
Document Shoryuken

### PR DESCRIPTION
**Pull request by kyrylo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,9 @@ Airbrake Changelog
 
 ### master
 
+* **IMPORTANT:** added Shoryuken integration
+  ([#669](https://github.com/airbrake/airbrake/pull/669))
```
Comment (phstc) [inline CHANGELOG.md]: 🤘

File: README.md [hunk 2]
```diff
@@ -379,6 +380,17 @@ require 'airbrake/delayed_job/plugin'
...
+### Shoryuken
+
+Simply `require` our error handler and you're done:
+
+```ruby
+require 'airbrake/shoryuken/error_handler'
+```
+
+If you required Shoryuken before Airbrake, then you don't even have to `require`
+anything manually and it should just work out-of-box.
```
Comment (phstc) [inline README.md]: ✨

File: CHANGELOG.md (hunk 1 above)
Comment (phstc) [inline CHANGELOG.md]: @kyrylo any ETA for releasing it? So I can remove my "patch"?

Comment (kyrylo) [inline CHANGELOG.md]: I'll release it this week. I plan to release airbrake-ruby v1.8.0 and this repo will need some adjustments.

Comment (phstc) [inline CHANGELOG.md]: @kyrylo sure. I'm using it from master now, do you see any possible issues with that?

Comment (kyrylo) [inline CHANGELOG.md]: Should be fine if you lock it to the current `HEAD`.

**Pull request merged**